src/rdkafka_mock_handlers.c

```diff
@@ -274,7 +274,6 @@ static int rd_kafka_mock_handle_Fetch(rd_kafka_mock_connection_t *mconn,
                                 err =
                                     rd_kafka_mock_partition_leader_epoch_check(
                                         mpart, CurrentLeaderEpoch);
-
```

Comment:
Since the comment is also there I have added that but somehow it does not show that , this is in response to another comment to add comment for all call sites of a function.